.dockerignore

```diff
@@ -0,0 +1,8 @@
+CHANGELOG.md
+README.md
+
+.github/
+mac/
+win-wpf/
+mattermost-plugin/
+website/
```

Comment:
probably we can also ad here `linux/` directory that is for generating the "linux desktop app"